Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -2148,6 +2155,7 @@
     mach_port_set_attributes
     mach_vm_copy
     mach_vm_map_external
+    (when (defined? 'mach_vm_range_create) mach_vm_range_create) ;; <rdar://105161083>
```

Comment:
I think also this sys call could only be allowed before the process has finished launching.

Reply:
I don't see a set of pre-launch mig routines in this sandbox, like there is for iOS. What's the best place to put such a conditional rule?

Reply:
Ah, good point! I think it could be put beneath the standard list of mig routines.

Reply:
I took a stab at making that work. My local build works properly on macOS, so I think it is correct. Thankfully, iOS was just adding to your existing "launch" collection of routines.